app/workers/organizations/update_organization_articles_paths_worker.rb

```diff
@@ -0,0 +1,15 @@
+module Organizations
+  class UpdateOrganizationArticlesPathsWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :high_priority
```

Comment:
Not sure if there should be a retry here

cc @mstruve

Reply:
We will retry automatically, without the option means we will do the full 25 retries rather than limiting that number.